app/src/main/kotlin/kt/school/starlord/utils/UrlUtils.kt

```diff
@@ -0,0 +1,12 @@
+package kt.school.starlord.utils
+
+import kt.school.starlord.BuildConfig
+
+/**
+ * Correcting URL if him not full.
+ * */
+fun urlCorrector(url: String) = if (url.startsWith("./")) {
```

Comment:
Создай конвертер для маппера, назови его `StringToUrlConverter` и перенеси отсюда всю логику в него.

Reply:
and use the mapper as like Document converter?

Reply:
StringUrlToStringUrl class? I think is not so nessesary to add converter from String to String.
It can be String to URL but is not converter logic it is instancing, can be logic dissonance

Reply:
Moved this method local into NetworkRepository class